website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -154,3 +154,7 @@ Workspaces may be configured to share their state with other workspaces within t
 ### Treat Archivist URLs as Secrets
 
 Terraform Cloud uses a blob storage service called Archivist for storing various pieces of customer data. Archivist URLs (which have the origin <https://archivist.terraform.io>) are returned by various Terraform Cloud APIs, such as the [state versions API](/terraform/cloud-docs/api-docs/state-versions#fetch-the-current-state-version-for-a-workspace). Unlike the rest of Terraform Cloud’s API, Archivist’s API does not require the user to submit a Bearer Token with each request. Instead, Archivist URLs contain a signed short term authorization token that is used to perform authorization checks. For this reason, Archivist URLs must be treated as secrets and should not be logged or shared.
+
+### Consider Using Dynamic Provider Credentials
+
+Storing static credentials in Terraform Cloud comes with increased risk in the event that these credentials are compromised. This is due to their long-lived nature, and the need to revoke these credentials in potentially many locations upon exposure. Using [Dynamic Provider Credentials](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/) eliminates the need to store credentials in Terraform Cloud and mitigates the aforementioned risks by generating new temporary credentials for each run.
```

Comment:
> Storing static credentials in Terraform Cloud comes with increased risk in the event that these credentials are compromised. This is due to their long-lived nature, and the need to revoke these credentials in potentially many locations upon exposure.

I'm being nitpicky on the language, but this feels more broad than we need to be. I think the specific scenario dynamic creds mitigates is a malicious user or compromised plan/apply (via injection or something else). Highlighting that dynamic credentials reduce the validity window of credentials so that in the event of a compromise of your plan or apply phase, those credentials are ephemeral and revoked at the end of the run, therefore reducing risk and exposure.

Reply:
That's fair! Maybe the following then?

```suggestion
Storing static credentials in Terraform Cloud comes with increased risk in the event that these credentials are exposed by a malicious user or a compromised plan or apply operation. This is due to their long-lived nature, and the need to revoke these credentials in potentially many locations upon exposure. Using [Dynamic Provider Credentials](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/) eliminates the need to store credentials in Terraform Cloud and reduces this potential risk and exposure by generating new temporary credentials for each run which expire after the run completes.
```

e.g., further scoping the first bit to add "by a malicious user or a compromised plan or apply operation."  and adjusting the end to be "and reduces this potential risk and exposure by generating new temporary credentials for each run which expire after the run completes."

Reply:
I think that sounds great!